### DISCUSSION TITLE
123云盘现在挂载不了，求解！

### DISCUSSION THREAD

**Discussion by kaixinxiaowangzi666** [AlistGo/alist#5532] [Q&A]
![Uploading 搜狗截图20231117143902.png…]()

**Comment by pongfcnkl** [comment]
不支持了，建议换一家薅

**Comment by yujinruiicu** [comment]
使用123的webdav来进行挂载

**Reply by derrickchang04** [reply]
怎么操作

**Reply by hight233** [reply]
要开 VIP ![Screenshot_2025-07-02-08-04-06-913_com.microsoft.emmx-edit.jpg](https://github.com/user-attachments/assets/0874bdc9-1ae2-48d5-88b4-e60ff5068234)

![Screenshot_2025-07-02-08-04-58-607_com.microsoft.emmx-edit.jpg](https://github.com/user-attachments/assets/54f520f0-e055-4e06-9820-d56824c537ef)

**Comment by hight233** [comment]
![Screenshot_2025-06-26-00-49-32-611_com.microsoft.emmx.jpg](https://github.com/user-attachments/assets/b49e3b8a-150e-446e-8d38-cea683a90cbf)

如图
